src/views/Returns.vue

```diff
@@ -0,0 +1,85 @@
+<template>
+    <ion-page>
+      <ion-header :translucent="true">
+        <ion-toolbar>
+          <ion-menu-button slot="start" />
+          <ion-title>{{ $t("Returns") }}</ion-title>
+        </ion-toolbar>
+      </ion-header>
+      <ion-content>
+        <main>
+          <ion-searchbar :placeholder="$t('Scan ASN to start receiving')"/>
+  
+          <ShipmentListItem v-for="shipment in shipments" :key="shipment.shipmentId" :shipment="shipment"/>
```

Comment:
Rename shipments to returns, and update the variable naming taking `returns` in consideration.

Reply:
Done sir